jdat_notebooks/environment.yml

```diff
@@ -23,10 +23,11 @@ dependencies:
   - pyregion
   - webbpsf>=0.8
   - scikit-learn
-  - lmfit
+  - lmfit>=0.9
   - specutils
   - dill
-
+  - dust-extinction
```

Comment:
```suggestion
  - dust_extinction
```

Reply:
Changed dust-extinction to dust_extinction